Makefile.toml

```diff
@@ -0,0 +1,107 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
+CARGO_MAKE_WORKSPACE_EMULATION = true
+CARGO_MAKE_CRATE_WORKSPACE_MEMBERS = [
+	"crt0stack",
+	"enarx-keep-sgx",
+	"enarx-keep-sgx-shim",
+	"enumerate",
+	"intel-types",
+	"iocuddle",
+	"iocuddle-sgx",
+	"memory",
+	"nolibc",
+	"payload",
+	"sev",
+	"sev-show",
+	"sev-types",
+	"sgx-crypto",
+	"sgx-show",
+	"sgx-types",
+	"span",
+	"testing",
+	"units",
+	"vdso",
+	"vmsyscall"
+]
+
+[tasks.audit]
+install_crate = "cargo-audit"
+command = "cargo"
+args = ["audit"]
+
+[tasks.deny]
+install_crate = "cargo-deny"
+command = "cargo"
+args = ["deny", "check", "licenses"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+
+[tasks.check-format]
+command = "cargo"
+args = ["fmt", "--", "--check"]
+
+[tasks.misc-lints-missing-docs]
+workspace = false
```

Comment:
What does `workspace = false` do here?

Reply:
This prevents `cargo-make` from descending into member crates. I removed that because I think our bash CI scripts glob the crates from the root-level directory.

Reply:
I think it means that it doesn't recurse into subprojects. Correct?